KomaMRIBase/src/timing/TimeStepCalculation.jl

```diff
@@ -0,0 +1,164 @@
+const MIN_RISE_TIME = 1e-10 #100*eps(1.0) #SmallestFloat
```

Comment:
```suggestion
const MIN_RISE_TIME = 1e-10
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/241/commits/6c2507cf6077ffacc5730b20c24184763332d45f